src/alcaramba/Drawable/Town/Town.gd

```diff
@@ -72,37 +76,95 @@ func _input(event):
 			if event.button_index == 1 && _placement_mode == 2:
 				# if cell not empty and cell is not starting tile and removed does not break connection
 				var cell_to_remove = get_cell(x, y)
-				if cell_to_remove != TileMap.INVALID_CELL && cell_to_remove != _starting_tile_id && is_tile_removable(x, y):
-					var removed_tile_id = remove_tile(x, y)
+				if cell_to_remove != TileMap.INVALID_CELL && cell_to_remove != _starting_tile_id && _is_tile_removable(x, y):
+					var removed_tile_id = remove_tile(x, y) # remove tile and return its id
```

Comment:
`removed_tile_id` is unused and this generates a warning. If this is intended, please rename to `_removed_tile_id`

Reply:
It will be needed later to send it to spare tiles